x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/util/SpatialCoordinateTypes.java

```diff
@@ -153,21 +134,19 @@ public long wkbAsLong(BytesRef wkb) {
         }
     }
 
-    public String wkbAsString(BytesRef wkb) {
-        Geometry geometry = WellKnownBinary.fromWKB(GeometryValidator.NOOP, false, wkb.bytes, wkb.offset, wkb.length);
-        if (geometry instanceof Point point) {
-            // WellKnownText.toWKT renders points slightly differently to SpatialPoint.toWKT
-            return pointAsString(pointAsPoint(point));
+    public BytesRef stringAsWKB(String string) {
+        // TODO: we should be able to transform WKT to WKB without building the geometry
+        // we should as well use different validator for cartesian and geo?
```

Comment:
I assume that since these utilities exist for converting internal, already indexed, types, validation is not required. But perhaps we do use these on the geometry literal path, so might like to check that at some point.

Reply:
Probably makes sense to check this during the work on geo_shape support.

Reply:
>But perhaps we do use these on the geometry literal path, so might like to check that at some point.